examples/resources/tlspc_firefly_config/resource.tf

```diff
@@ -0,0 +1,6 @@
+resource "tlspc_firefly_config" "ff_config" {
+  name             = "Firefly Config"
+  subca_provider   = resource.tlspc_firefly_subca.subca.id
+  service_accounts = [resource.tlspc_service_account.sa.id]
+  policies         = [resource.tlspc_firefly_policy.ff_policy.id]
```

Comment:
Some of these things are coming in future PRs, but this works as a feature (you could just provide shas), and this is just an example. Made more sense to me to start with the resource we want to have, and then use that to discover the things we need to implement to support it.